Doc/library/asyncio-eventloop.rst

```diff
@@ -214,7 +214,8 @@ Scheduling callbacks
 .. method:: loop.call_soon_threadsafe(callback, *args, context=None)
 
    A thread-safe variant of :meth:`call_soon`.  Must be used to
-   schedule callbacks *from another thread*.
+   schedule callbacks *from another thread*. If the event loop has been
+   closed, :exc:`RuntimeError` is raised.
```

Comment:
```suggestion
   schedule callbacks *from another thread*.
   
   Raises :exc:`RuntimeError` if called on a loop that's been closed.
   This can happen on a secondary thread when the main application is
   shutting down.
```